### PR TITLE
CDAP-7325 Add CDAP classes in spark classpath in the beginning.

### PR DESCRIPTION
**Pull request by sagarkapare**
JIRA: https://issues.cask.co/browse/CDAP-7325
Release build is running here - https://builds.cask.co/browse/CDAP-RUT188-1

### CODE REVIEW

File: cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java [hunk 1]
```diff
@@ -474,24 +464,29 @@ private static String getCdapCommonJarName() {
...
-      String extraClassPath = joiner.join(Paths.get("$PWD", CDAP_LAUNCHER_JAR), cdapCommonJarPath,
+      String classpath = joiner.join(jarFiles);
+      String extraClassPath = joiner.join(Paths.get("$PWD", CDAP_LAUNCHER_JAR), classpath,
                                           Paths.get("$PWD", CDAP_SPARK_JAR, "lib", "*"));
       if (logbackJarName != null) {
         extraClassPath = logbackJarName + File.pathSeparator + extraClassPath;
       }
 
-      LOG.debug("Setting spark.driver.extraClassPath and spark.executor.extraClassPath to {}.", extraClassPath);
+      LOG.info("Setting spark.driver.extraClassPath and spark.executor.extraClassPath to {}.", extraClassPath);
```
Comment (chtyim) [inline cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java]: Why change the log level?

Comment (sagarkapare) [reply cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java]: Added it for debugging. Reverted back to debug.

**Comment by sagarkapare** [comment]
@chtyim Addressed comments. Please take a look again. Thanks!
Build is green here - https://builds.cask.co/browse/CDAP-RUT188-1.

**Comment by chtyim** [comment]
The changes looks fine. Do we need to do the same for map reduce?

**Comment by sagarkapare** [comment]
@chtyim MapReduce changes are already done. https://github.com/caskdata/cdap/blob/develop/cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java#L284.

**Comment by gokulavasan** [comment]
LGTM 👍

**Comment by sagarkapare** [comment]
Build is green https://builds.cask.co/browse/CDAP-RUT188-3. Squashing and merging. @chtyim Please let me know if you have any additional comments. Will address them in separate PR. Thanks!

**Pull request merged**